<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualEventScreen</class>
 <widget class="QWidget" name="ManualEventScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>356</width>
        <height>636</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="EventTypePanel" name="eventPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="whenTab">
          <attribute name="title">
           <string>When?</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="CalendarWidget" name="startCalendar" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>270</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>270</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="horizontalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Time:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTimeEdit" name="timeEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Duration (Min):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="durationBox"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>234</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="notesTab">
          <attribute name="title">
           <string>About</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QStackedWidget" name="notesStack">
             <widget class="QWidget" name="feedingPage">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPlainTextEdit" name="notesEdit"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="medicinePage">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Drug Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="drugEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Notes:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="drugNotesEdit"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="scheduleTab">
          <attribute name="title">
           <string>Schedule</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>End date:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="CalendarWidget" name="endCalendar" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>270</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>270</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Repeat in (Min.):</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="repeatMinBox">
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>202</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventTypePanel</class>
   <extends>QWidget</extends>
   <header>eventtypepanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QWidget</extends>
   <header>calendarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
